experiment_management/bundle_integrate_mlflow.ipynb

```diff
@@ -161,7 +163,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "monai.bundle.download(name=\"spleen_ct_segmentation\", version=\"0.4.2\", bundle_dir=\"./\")"
+    "if not os.path.exists(\"./spleen_ct_segmentation\"):\n",
+    "    monai.bundle.download(name=\"spleen_ct_segmentation\", bundle_dir=\"./\")"
```

Comment:
How about we include the download progress output so that the user can know which bundle version was used?

Reply:
Hi @mingxin-zheng ,
    Currently the we can only download the old version '0.3.7' bundle from NGC, which doesn't match the new bundle workflow commandline in this tutorial. So I manually download the `0.4.2` bundle. If we include the download bundle version here, I think it maybe confuse for the users.
```
!python -m monai.bundle run \
    --config_file configs/train.json \
    --bundle_root ./ \
    --dataset_dir /workspace/data/Task09_Spleen \
    --train#trainer#max_epochs 10 \
    --tracking "mlflow"
``` 

Thanks,
Bin

Reply:
Is it already 0.4.2 on GitHub Model Zoo? I think the NGC doesn't apply here in GitHub tutorials

Reply:
Please take this as an optional suggestion feel free to ignore.